src/libcollections/str.rs

```diff
@@ -599,7 +599,7 @@ impl str {
     /// # #![feature(str_char, core)]
     /// use std::str::CharRange;
     ///
-    /// let s = "中华Việt Nam";
+    /// let s = "中华Việt Nam";
```

Comment:
Might depend on the browser? Since it renders the diacritics of the `e` on top of the `t` in the after version, I think the diff rendering is broken here.